packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -144,56 +128,138 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         Cypress.action('cy:enqueue:command', attrs)
       }
 
-      const updateCommand = (details) => {
-        if (details.logAdded) {
-          const attrs = details.logAdded
+      const updateCommand = ({ id, end }) => {
+        if (end) {
+          const command = commands[id]
+
+          if (command) {
+            delete commands[id]
+            command.deferred.resolve()
+          }
+        }
+      }
```

Comment:
Sorry I didn't notice this earlier, but this should be changed to `endCommand` and the event to `command:end`. Then we can get rid of the `end` property and the `if` condition here.

Reply:
No worries! I renamed the event to `command:end` and only kept the ID on destructure.  Figured we can keep the destructure pattern as we like to send other properties sometimes to debug